package.json

```diff
@@ -29,49 +29,64 @@
     "duplicate-arguments-array": false
   },
   "scripts": {
-    "test": "gulp test",
-    "prepublish": "bower install && not-in-install && gulp build || in-install"
+    "test": "gulp test"
   },
   "dependencies": {
     "@slack/client": "^3.5.4",
     "@devicefarmer/adbkit": "^2.11.2",
     "@devicefarmer/adbkit-apkreader": "^3.2.1",
     "@devicefarmer/adbkit-monkey": "^1.0.1",
+    "@devicefarmer/minicap-prebuilt": "^2.4.0",
+    "@devicefarmer/minitouch-prebuilt": "^1.3.0",
+    "@devicefarmer/stf-appstore-db": "^1.0.0",
+    "@devicefarmer/stf-browser-db": "^1.0.2",
+    "@devicefarmer/stf-device-db": "^1.2.0",
+    "@devicefarmer/stf-syrup": "^1.0.1",
+    "@devicefarmer/stf-wiki": "^1.0.0",
+    "@julusian/jpeg-turbo": "^0.5.4",
+    "@slack/client": "^3.5.4",
```

Comment:
`@slack/client` package added twice which cause the `package.json` file to be updated during installation